src/app/models/question.model.js

```diff
@@ -4,6 +4,7 @@ let questionSchema = mongoose.Schema({
   text : { type: String, required: true, maxlength: 1000 },
   user : { type: mongoose.Schema.Types.ObjectId, ref: 'User', required: true },
   answered: Boolean,
```

Comment:
Is this property still necessary (looking if `answeredAt` is set would give the same info)?